cmd/definitions/tmpl/service.tmpl

```diff
@@ -103,7 +103,7 @@ func setStorageSystemMetadata(s *StorageMeta, sm StorageSystemMetadata) {
 }
 
 {{- range $_, $v := .Pairs }}
-{{- if not $v.Global }}
+{{- if and (not $v.Global) (not (hasPrefix $v.Name "enable_")) }}
```

Comment:
I removed `WithEnableXxx()` for features as we can use `<type>.WithStorageFeatures(<type>.StorageFeatures{Xxx: true, })` during initialize. I'm not sure whether it's necessary.

Reply:
I think it's OK to allow users to enable a feature via `WithEableXxx()`.

Reply:
OK, I'll roll back this change.

Reply:
For bool types, maybe we can generate `WithEnableXxxx()` directly? With the internal value set to `true`.